restler/unit_tests/test_restler_settings.py

```diff
@@ -235,6 +235,30 @@ def test_checkers_in_settings(self):
         self.assertEqual(None, settings.get_checker_arg('leakage_rule_mode', 'mode'))
         self.assertEqual(None, settings.get_checker_arg('resource_hierarchy_rule_mode', 'mode'))
 
+    def test_validate_string_values_in_candidate_values(self):
+        candidate_values = primitives.CandidateValuesPool()
```

Comment:
>candidate_values [](http://example.com/codeflow?start=8&length=16)

This is failing since the Settings object is not initialized.  Each test needs to have the following initialization: ```        settings_file = {}
_ = RestlerSettings(settings_file)
```